types/puppeteer/index.d.ts

```diff
@@ -2029,6 +2030,12 @@ export interface Target {
 }
 
 export interface LaunchOptions extends ChromeArgOptions, BrowserOptions, Timeoutable {
+  /**
+   * Which browser to launch.
+   * At this time, this is either `chrome` or `firefox`. See also `PUPPETEER_PRODUCT`.
+   * @default 'chrome'
+   */
+  product?: 'chrome' | 'firefox';
```

Comment:
Please move this to a union string. eg. `ProductType = 'chrome' | 'firefox';`

Reply:
@SimonSchick 
Done, amended!